openverse_catalog/dags/providers/provider_dag_factory.py

```diff
@@ -208,7 +214,9 @@ def create_provider_api_workflow(
                             generate_filenames.task_id, f"{media_type}_tsv"
                         ),
                         "s3_bucket": OPENVERSE_BUCKET,
-                        "s3_prefix": f"{media_type}/{provider_name}",
+                        "s3_prefix": DATE_PARTITION_ARG_TEMPLATE.format(
+                            media_type, provider_name
```

Comment:
In case you accept the suggestion above to name the holes:

```suggestion
                            media_type=media_type,
                            provider_name=provider_type,
```

Reply:
Thanks!